### ISSUE TITLE
Include "Custom Alt Text" in Advanced Settings

### ISSUE THREAD

**Issue by TheSharpG** [elementor/elementor#9624]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Enhanced SEO without creating duplicate image content.

This is why custom alt text is an important Google ranking signal: https://yoast.com/image-seo-alt-tag-and-title-tag-optimization/

**Describe the solution you'd like**
Allow for "custom alt text" for images in the advanced settings of an image.

This custom alt text would override the alt text provided in the media library.

Putting it in "advanced" would also prevent users from getting confused, which I read was an issue in the past and what got this feature removed in the first place. Source: https://github.com/elementor/elementor/issues/1053

**Describe alternatives you've considered**
Right now, the only way to create custom alt text for the same image is to upload the image twice or more. The problem with this is that it creates duplicate image content, which is detrimental for SEO.

Some have suggested the WP Meta SEO plugin as a way around this, but I have not found this to work- and using this plugin would also conflict with the Yoast plugin, which I find to be preferable.

**Additional context**
In the past I've provided SEO consulting for Microsoft, Proctor & Gamble, and ADP, and am happy to provide any advice to make Elementor better! It is a great tool.

**Comment by vekman-design** [comment]
Full support for this! Just purchased Pro and this is almost the only main feature I'm still missing.

**Comment by rodfilardi** [comment]
This is really a basic and super important item. It is unbelievable that a plugin like this, so popular in the SEO community, does not have this functionality.

**Comment by romain-d** [comment]
Hello,

Any news on this add request?
My customers would like to have images with different alt and therefore do not touch the general alt of the image.
It would be nice to be able to do this in Elementor directly for each image

**Comment by BetterOutcomesAdvocacy** [comment]
The current implementation has a significant impact on the vision-impaired user as vision-impaired crawlers such as Google. 

I think I only need to point out that the current setup is discriminating against people with vision impairment to compel you to rethink this. 

https://www.visionaustralia.org/services/digital-access/document-accessibility-toolbar/user-guide
 
"Providing a description (a text alternative) of the images and objects in your document is important for those individuals who cannot see the images you have included. The text or description you include will be read aloud by a screen reader. The text alternative must convey the purpose of the image or object."

**Comment by DavidGMiles** [comment]
This is a massive omission - if I need two images that are the same and only differ by ALT text, then I have to upload the images twice into the media library varying only the ALT text - plus we really need these to be driven by dynamic tags to be 100% useful. This is a crazy decision that flies in the face of SEO and accessibility

**Comment by wordxpression** [comment]
I agree it's a huge omission. Please add it asap in a future version.

**Comment by sharmarch** [comment]
Please add this functionality! Duplicate images in the media library to accomplish the same is totally inefficient.

**Comment by Antotna** [comment]
I agree. It would be very useful (and it's really needed). Thanks

**Comment by DynWeb-John** [comment]
The new block editor for wordpress has this functionality in it. As such we can now do the same in Elementor by using the "WordPress Image" widget instead of the "Elementor image" widget. If you want to style the image, you will need to use CSS (in advanced settings), but you can use the Edit image button to update the images Alt text in a localised version of the image, without updating the media library default alt text.

**Comment by mathematicswizard** [comment]
This is a much needed feature. I'm actually quite amazed Elementor still haven't added it.

**Comment by tomas-eklund** [comment]
I came here to request the same thing. The Alt and Title fields of an image widget needs to be overridden occasionally. Also I need them to be dynamic, so that I can use the image widget in templates and override Alt and Title dynamically.

**Comment by SilverFoxA** [comment]
+1

**Comment by mvogelenzang** [comment]
Same here. I am using thumbnails which I want to have dynamic text in, so obviously I cant just use whatever alt text I set as default in the WP Media Gallery.

**Comment by CarterStevens1** [comment]
I agree that this should be added back in as it will 100% be beneficial for SEO

**Comment by DavidGMiles** [comment]
I have recently worked on a multi-lingual site - it supports three languages. To achieve ALT text in the user's language doing it the Elementor way would have required me to upload each image three times - that is madness - if I can write an extender that allows for the addition of custom ALT and Title fields, then I am damned sure that the Elementor developers can do so too! This is a ridiculous thing to have left to languish for over three years

**Comment by xamanek** [comment]
Hello, i find this plugin and works good but only for alt text, no for the title

https://wordpress.org/plugins/custom-alt-text-for-elementor-images/
